tox.ini

```
@@ -50,6 +50,13 @@ deps =
     bandit
 commands = bandit -r metadata_backend/
 
+[testenv:documents]
```

we already have `testenv:docs` maybe we can name it like this (to avoid confusion):
```suggestion
[testenv:spelling]
```